apps/admin/backend/src/store.test.ts

```diff
@@ -56,700 +52,25 @@ test('assert election exists', () => {
   );
 });
 
-test('add a CVR file', async () => {
```

Comment:
Previously, we were testing a lot of CVR actions in `store.test.ts` and then testing them again at a higher level in `app.test.ts`. Now we're mostly only testing it in `app.cvr_files.test.ts` (or `app.cdf_cvr_files` at this point in the PR), so there's less testing overall but the coverage is pretty much the same. I'm for testing at the higher level (which uses the store) and not belaboring the endpoints here.

Reply:
I'm into it